Analyses/src/StepPointMC1stHitDumper_module.cc

```diff
@@ -195,7 +215,7 @@ namespace mu2e {
         // std::cout << event.id() << ": " << stepPoints.at(i) << std::endl;
       // }
       if (stepPoints.size()) {
-        hit_ = VDHit(toff_, stepPoints.at(0));
+        hit_ = VDHit(toff_, event, stepPoints.at(0));
```

Comment:
Hi Dave,

The StepPointMC1stHitDumper module is just an analyzer module that makes a simple ntuple from the first step point MC. I found it handy to make simple plots for diagnostics purposes.
The upper momentum cutoff is done in the filter module: FilterStepPointMomentum.
The upper momentum cut of 0.5 GeV removes all the straight tracks and it reduces the size of the files by about a factor 10. It will be useful to have this functionality for a large scale cosmics MC production.

I've submitted these changes in one commit, and they are completely unrelated.
Sorry, if it adds confusion.
Would it help to separate commits in the future?